articles/virtual-machines/disks-redundancy.md

```diff
@@ -31,23 +31,38 @@ Zone-redundant storage (ZRS) replicates your Azure managed disk synchronously ac
 
 ZRS disks allow you to recover from failures in availability zones. If an entire zone went down, a ZRS disk can be attached to a VM in a different zone. You can also use ZRS disks as a shared disk to provide improved availability for clustered or distributed applications like SQL FCI, SAP ASCS/SCS, or GFS2. You can attach a shared ZRS disk to primary and secondary VMs in different zones to take advantage of both ZRS and [Availability Zones](../availability-zones/az-overview.md). If your primary zone fails, you can quickly fail over to the secondary VM using [SCSI persistent reservation](disks-shared-enable.md#supported-scsi-pr-commands).
 
+
+### Billing implications
+
+For details see the [Azure pricing page](https://azure.microsoft.com/pricing/details/managed-disks/).
+
+### Comparison with other disk types
+
+Except for more write latency, disks using ZRS are identical to disks using LRS. They have the same performance targets. We recommend you to conduct [disk-benchmarking](disks-benchmarks.md) to simulate the workload of your application for comparing the latency between the LRS and ZRS disks. 
+
 ### Limitations
 
 During the preview, ZRS for managed disks has the following restrictions:
 
 - Only supported with premium solid-state drives (SSD) and standard SSDs.
-- Currently available only in the EastUS2EUAP region.
+- Currently available only in the West US 2, West Europe, North Europe and France Central regions.
 - ZRS disks can only be created with Azure Resource Manager templates using the `2020-12-01` API.
 
-Sign up for the preview [here](https://aka.ms/ZRSDisksPreviewSignUp).
+## Prerequisites
 
-### Billing implications
+You must enable the feature for your subscription. Please follow the steps below to enable the feature for your subscription:
 
-For details see the [Azure pricing page](https://azure.microsoft.com/pricing/details/managed-disks/).
+1.	Execute the following command to register the feature for your subscription
 
-### Comparison with other disk types
+    ```powershell
+     Register-AzProviderFeature -FeatureName "SsdZrsManagedDisks" -ProviderNamespace "Microsoft.Compute" 
+    ```
 
-Except for more write latency, disks using ZRS are identical to disks using LRS. They have the same performance targets. We recommend you to conduct [disk-benchmarking](disks-benchmarks.md) to simulate the workload of your application for comparing the latency between the LRS and ZRS disks. 
+2.	Please check that the registration state is Registered (takes a few minutes) using the command below before trying out the feature.
```

Comment:
```suggestion
1.	Confirm that the registration state is **Registered** (it may take a few minutes) using the following command before trying out the feature.
```